circuit/src/queryBuilder/dataSubquery/configLimitManager.ts

```diff
@@ -0,0 +1,177 @@
+import {
```

Comment:
Assuming `dataSubquery/` is all copied from `sdk-core`, did not carefully review